README.md

```diff
@@ -22,7 +22,7 @@ USAGE:
 ARGUMENTS:
     <GITHUB-API-KEY>    The GitHub API key to use.
                         The API key must have the following permissions: 
-                        public_repo, delete_packages   
+                        public_repo, delete_repo, delete_packages   
```

Comment:
That is actually a typo. `delete_packages` should be deleted as well.

Reply:
That's what I figured but wasn't sure about!